luaunit.lua

```diff
@@ -175,6 +175,22 @@ local function sortedPairs(tbl)
 end
 M.private.sortedPairs = sortedPairs
 
+math.randomseed(os.clock()*100000000000)
+
+local function randomizeTable( t )
+    -- Shuffle the table t
+    local rand = math.random 
+    assert( t, "shuffleTable() expected a table, got nil" )
+    local iterations = #t
+    local j
+    
+    for i = iterations, 2, -1 do
+        j = rand(i)
+        t[i], t[j] = t[j], t[i]
+    end
```

Comment:
Optional: `math.random(i)` may return `i`, so maybe it would be useful to avoid the swapping if `j == i`?
```lua
        if j ~= i then
            t[i], t[j] = t[j], t[i]
        end
```